connect/cli/plugins/shared/constants.py

```diff
@@ -6,3 +6,110 @@
     'comment',
     'editor',
 ]
+
+ITEMS_COLS_HEADERS = {
+    'A': 'ID',
+    'B': 'MPN',
+    'C': 'Action',
+    'D': 'Name',
+    'E': 'Description',
+    'F': 'Type',
+    'G': 'Precision',
+    'H': 'Unit',
+    'I': 'Billing Period',
+    'J': 'Commitment',
+    'K': 'Status',
+    'L': 'Created',
+    'M': 'Modified',
+}
+
+PARAMS_COLS_HEADERS = {
+    'A': 'Verbose ID',
+    'B': 'ID',
+    'C': 'Action',
+    'D': 'Title',
+    'E': 'Description',
+    'F': 'Phase',
+    'G': 'Scope',
+    'H': 'Type',
+    'I': 'Required',
+    'J': 'Unique',
+    'K': 'Hidden',
+    'L': 'JSON Properties',
+    'M': 'Created',
+    'N': 'Modified',
+}
+
+MEDIA_COLS_HEADERS = {
+    'A': 'Position',
+    'B': 'ID',
+    'C': 'Action',
+    'D': 'Type',
+    'E': 'Image File',
+    'F': 'Video URL Location',
+}
+
+CAPABILITIES_COLS_HEADERS = {
+    'A': 'Capability',
+    'B': 'Action',
+    'C': 'Value',
+}
+
+STATIC_LINK_HEADERS = {
+    'A': 'Type',
+    'B': 'Title',
+    'C': 'Action',
+    'D': 'Url',
+}
+
+TEMPLATES_HEADERS = {
+    'A': 'ID',
+    'B': 'Title',
+    'C': 'Action',
+    'D': 'Scope',
+    'E': 'Type',
+    'F': 'Content',
+    'G': 'Created',
+    'H': 'Modified',
+}
+
+CONFIGURATION_HEADERS = {
+    'A': 'ID',
+    'B': 'Parameter',
+    'C': 'Scope',
+    'D': 'Action',
+    'E': 'Item ID',
+    'F': 'Item Name',
+    'G': 'Marketplace ID',
+    'H': 'Marketplace Name',
+    'I': 'Value',
+}
+
+ACTIONS_HEADERS = {
+    'A': 'Verbose ID',
+    'B': 'ID',
+    'C': 'Action',
+    'D': 'Name',
+    'E': 'Title',
+    'F': 'Description',
+    'G': 'Scope',
+    'H': 'Created',
+    'I': 'Modified',
+}
```

Comment:
should all these constants be located in `shared`? in the end, aren't they used just in the `product`plugin?

Reply:
Well in this case may not use this directly, but it is use also by the helper `get_col_headers_by_ws_type` that is use by `ProductSynchronizer` and finally this is use by `TranslationsSynchronizer` for inheritance, so I need this to have it here.